tests/integration/test_script_runner.py

```diff
@@ -15,12 +15,27 @@ class BackendDependencies(enum.Enum):
     MSSQL = "MSSQL"
     PANDAS = "PANDAS"
     POSTGRESQL = "POSTGRESQL"
+    REDSHIFT = "REDSHIFT"
```

Comment:
Good catch. Would you please ensure that if a user passes the `--no-sqlalchemy` flag to pytest that the redshift tests are skipped? I think that requires a change to this file.

Reply:
added redshift logic to `_check_for_skipped_tests()` function below.